web/src/ee/features/evals/server/router.ts

```diff
@@ -42,6 +43,27 @@ export const CreateEvalTemplate = z.object({
     .default(EvalReferencedEvaluators.PERSIST),
 });
 
+const CreateEvalJobSchema = z.object({
+  projectId: z.string(),
+  evalTemplateId: z.string(),
+  scoreName: z.string().min(1),
+  target: z.string(),
+  filter: z.array(singleFilter).nullable(), // re-using the filter type from the tables
+  mapping: z.array(variableMapping),
+  sampling: z.number().gte(0).lte(1),
```

Comment:
Nit: `sampling` should be `sampling: z.number().gt(0).lte(1),`, as sampling rate of 0 does not make sense. I believe we control for this on the FE already

Reply:
Done, thanks 👍🏾